system-contracts/bootloader/bootloader.yul

```diff
@@ -301,11 +301,12 @@ object "Bootloader" {
                 ret := mul(add(MAX_TRANSACTIONS_IN_BATCH(), 1), TX_OPERATOR_L2_BLOCK_INFO_SLOT_SIZE())
             }
 
-            /// @dev The last processed block number.
+            /// @dev The slot right after the last slot of the last processed block number.
```

Comment:
why is it the right after one? I think it is that one exactly, no? similar below